### ISSUE TITLE
/afk bug for scenarios

### ISSUE THREAD

**Issue by Skelltt** [WarEmu/WarBugs#20818]
**Expected behavior and actual behavior:**
Expected: You should not be able to join a scenario with /afk flag on. Or you should at least get thrown out at the start a scenario with /afk flag on.

Actual: You can join a scenario with /afk flag on. You will not get thrown out the way if you join in later.

### **What does this mean?**

When you join a scenario with a /afk flag on, there are two instances that can happen:
1. You join before the scenario starts, the starting countdown goes down, you will get thrown out of the scenario if your /afk flag is still on.
2. You join the scenario later, after the starting countdown is over. The game thinks it boots you out of the scenario, but it does not really.
The game thinks it boots you out with /afk flag on if you join the game later. And you will receive this message:

![grafik](https://user-images.githubusercontent.com/76249403/228668588-d5f24784-695d-4a22-8307-01f8e77312e2.png)

But it actually does not throw you out. I was able to play in the scenario (check scenario killboard where i got a kill. I was not shown in the scoreboard afterwards though).
Any kills i got it acted as if i was in open rvr, which means i got renown and crests (which i probably? didnt share with the people in the sc, since i was not in a group).

Since the game thinks it threw me out of the sc, it filled my place with another player rejoining, even though i was still in the sc (check killboard picture).
That means it is possible for a faction to have more than 12 people active in a scenario with this "method".

**Steps to reproduce the problem:**
1. Queue for scenario with /afk on.
2. Join the scenario later, at a point where the countdown timer is already over.
3. The game does not throw you out of the scenario.

If you put on /afk DURING a scenario, the game will still throw you out correctly.

**Testing Screenshots/Videos/Evidences (always needed):**

Here you can see me having a kill on the killboard in an sc.
The same sc table shows me actually not being in the sc, since the game thinks it threw me out.

![bug report1](https://user-images.githubusercontent.com/76249403/228668139-8471d480-2253-4271-850d-0e95e909eb4e.jpg)
![bug report 2](https://user-images.githubusercontent.com/76249403/228668154-2d511c30-1548-4bbb-82f7-5e0d6cdaeec7.jpg)

### Sidenote
NOT TESTED: This bug is most likely also reproducable as a duo or full group even. The prerequites will probably be the same (queue with /afk flag on and join at a point where the countdown timer at the start of the scenario is already over).

**Comment by lilimx** [comment]
Wow... This is a nice report.... But it was already reported here: #20218

**Comment by Skelltt** [comment]
Yes the other report is the same topic, but reporting the issue the other way around. I think mine goes a little bit more in-depht about it.

Basically it boils down to this:

The game doesn't correctly kick you if you join a scenario with /afk flag on AFTER the countdown timer has run down.
The countdown timer (you are in the scenario waiting for it to start) acts as a checkpoint to check whether your /afk is on or not. It throws you out correctly if it is on (since you shouldn't be afk in a scenario). If you join in later after that checkpoint is over, you should be thrown out of the sc immediately the instance you join in (with /afk flag on). If you do that, the game tells you, that you have been thrown out and you receive a quitter debuff, but you are not actually out. It's the "kicking you out of the sc part" that bugged currently.

**Comment by TilliRage** [comment]
I had the same issue. Stuck in the scenario got no destro to kill me. Used /afk and got kicked out from sc. Waited 10m quitter debug and que for a sc again. It pops and I join. Chat gives me this message with another quitter debuff. Was able to play but cannot see objectives or scoreboard. 
![image](https://user-images.githubusercontent.com/66213697/234574562-e5d55582-9b3d-4015-bfa6-12c0f383bd70.png)
